### DISCUSSION TITLE
Generate Twig extension

### DISCUSSION THREAD

**Discussion by wsydney76** [craftcms/generator#8] [Ideas] [answered]
Would be great to be able to generate a Twig extension with examples how a custom filter/function can look like. (Optionally globals?)

Thanks for considering.

/Mel

**Comment by brandonkelly** [comment, marked as answer]
Good idea! Just released Generator 1.1.0 with a Twig Extension generator:

```sh
php craft make twig-extension
```

<img width="1170" alt="A terminal window showing the new `make twig-extension` command in action, with the generated Twig extension class opened in PhpStorm in the background." src="https://user-images.githubusercontent.com/47792/211078786-2ec5080f-745d-4a38-a63b-bc3a30a3e024.png">
